src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -142,28 +142,28 @@ public Hardware() {
         if (!comp)
             return;
         if (CLIMB_ENABLED) {
-            climbMotorDynamic = new WPI_TalonFX(CLIMB_DYNAMIC_MOTOR);
+            // climbMotorDynamic = new WPI_TalonFX(CLIMB_DYNAMIC_MOTOR);
```

Comment:
This causes simulation to immediately crash. Need to actually update the climb subsystem to handle the optional dynamic arm vs. just commenting out.

Reply:
Still crashing...

Reply:
fixed